pkg/storage/image.go

```diff
@@ -285,11 +328,19 @@ func GetImageService(store storage.Store, defaultTransport string, insecureRegis
 		}
 	}
 
+	if len(additionalRegistries) == 0 {
+		// TODO(runcom): make docker.io a CONST
+		additionalRegistries = []string{"docker.io"}
```

Comment:
Perhaps the flag should just be registries, and we allow users to add a list of registries, with docker.io as the default.